assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/deployment.yaml

```diff
@@ -115,14 +115,6 @@ rules:
       - get
       - create
       - update
-  # Allow using PSP
-  - apiGroups: ["policy"]
```

Comment:
Is it an upstream change? What policy will be used then?

Reply:
This is an upstream change and it will use privileged PSP. BTW we never shipped any PSP named `calico-kube-controllers`.